sdk/resource/resource.go

```diff
@@ -168,7 +193,10 @@ func Merge(a, b *Resource) (*Resource, error) {
 	}
 
 	// Merge the schema URL.
-	var schemaURL string
```

Comment:
Nit: We are no longer returning early, could this switch statement be at the end of the function and use returns?

something like:
```
    switch {
    case a.schemaURL == "":
		return NewWithAttributes(b.schemaURL, combine...), nil
	case b.schemaURL == "":
		return NewWithAttributes(a.schemaURL, combine...), nil
	case a.schemaURL == b.schemaURL:
		return NewWithAttributes(a.schemaURL, combine...), nil
	default:
        return NewWithAttributes("", combine...), nil
```

Reply:
Updated.